### ISSUE TITLE
Flux forced - varying heat flux by a certain factor

### ISSUE THREAD

**Issue by dhruvbhagtani** [dhruvbhagtani/sfc-perturbation-expts#39]
We have successfully created a set of simulations where changing one doesn't affect the other. In the following issue, we discuss a set of experiments where we change the heat flux everywhere by a certain factor, keeping the wind stress the same, as shown in the animation. We run the control, 0.5xH (0.5 times the heat flux compared to control) and 1.5xH (1.5 times the heat flux compared to control) for 100 years each.

https://user-images.githubusercontent.com/29700296/153521994-b8dd0802-7357-45d2-8bef-3daa2f006cda.mp4

**Comment by dhruvbhagtani** [comment]
All results can be found here: 
https://github.com/dhruvbhagtani/varying-surface-forcing/blob/main/025deg_flux_expts/Flux_vary_heat.ipynb

**Comment by dhruvbhagtani** [comment]
We observe a consistent increase in gyre strength with heat forcing as well, which is consistent with two results/intuitions:
1. Hogg and Gayen (2020) suggest that gyres could be driven by buoyancy forcing alone.
2. By increasing meridional contrast of surface heat flux, the ocean will require mechanisms to transport heat from the equator to the poles, and one of these mechanisms could be ocean gyres, as shown in the figure.

![Github_upload 003](https://user-images.githubusercontent.com/29700296/153522154-9104ef73-d025-4540-84bd-3533ed6e0dcf.jpeg)

![Github_upload 002](https://user-images.githubusercontent.com/29700296/153539902-cd8ce132-1d6a-4fd2-a93a-f90e3537fac2.jpeg)

**Comment by dhruvbhagtani** [comment]
The red contours are same for all the cases - it shows the extent of the North Atlantic subtropical gyre in the control run. We superimpose this contour on all cases to understand the spatial differences between gyres of different experiments.

**Comment by dhruvbhagtani** [comment]
Here are two videos for NA gyre showing how the top (0-1000 m) and deep (1500-5000 m) ocean evolve with time. There are a lot of interesting results from these plots:
1. The gyre strength increases a lot very quickly in the first 10 years, especially in the WBC where we have an increased heat loss. 
2. After 10 years, we see two things: (i) The subpolar gyre spills over to the western boundary of the subtropical gyre, and the WBC of the gyre goes to the NADW formation regions, and there are some dynamics involved which ultimately reduce the gyre strength over time.

https://user-images.githubusercontent.com/29700296/158489214-d2f101ae-bb37-4912-ada9-754ad3f3a5b6.mp4


https://user-images.githubusercontent.com/29700296/158489222-a44c4980-6aa3-497f-8fd7-47b4675ef434.mp4

But the fact that the gyre strength increases so much says a lot about the influence of heat flux forcing.

**Comment by AndyHoggANU** [comment]
Wow, this is a big response...
There seems to be an interesting effect in the second half of the run, where the deep gyre spins up, apparently at the expense of the surface gyre? Is that a subpolar gyre influence?

**Comment by dhruvbhagtani** [comment]
Yes, a very big response indeed!

That's a good point. I don't know yet, I'll have a look at how stratification evolves, maybe that'll help. I'll also post a similar video for the control and 0.5x heat case to compare as soon as they are prepared.

**Comment by rmholmes** [comment]
Cool. The deep and surface responses do seem largely opposite. Although there is quite a prominent 15-30 year oscillation in the (surface) strength of the subpolar gyre not reflected at depth which is interesting.

**Comment by dhruvbhagtani** [comment]
Oh, I didn't even realise such a prominent oscillation! Thanks - very interesting.

**Comment by dhruvbhagtani** [comment]
@rmholmes correct me if I'm wrong, but I can observe this oscillation in the control run too. Don't know why that is, since we don't have any inter annual variations in forcing.

https://user-images.githubusercontent.com/29700296/158521660-556f419f-91dd-4c30-ba2d-f4aa6e71d2ab.mp4


https://user-images.githubusercontent.com/29700296/158521645-13bcc80e-9162-432f-ab68-69235b1adf11.mp4

**Comment by rmholmes** [comment]
Do you mean specifically in the bottom video toward the end of the run? There is definitely some interesting internal variability going on in there. It would be nice to see a time series of the gyre transport in the Labrador sea.

However, we may be getting into a bit of a can of worms here. Internal variability in the AMOC and deep water formation in the North Atlantic is complicated and model dependent. We need to consider it - but probably only to the extent of whether our signal (the difference between experiments) is larger than the noise (the internal variability).

**Comment by dhruvbhagtani** [comment]
No, I can see those oscillations from the beginning itself. The spatial structure is different but I think that is due to forcing. I'll plot the time series and compare with other perturbation experiments. 

You're right: after looking at the control run, I am not too much worried about the oscillations as they seem to be model dependent rather than forcing dependent (to a large extent, at least). But I can't say for sure.
